Failing to compile repair_polygon_soup_example.cpp

New user trying to use CGAL for a project. Specifically trying compile and use mesh repair examples.

## Issue Details

Compilation error: Issue compiling calls that are made to "internal_np":
```
In file included from /Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:4:
/usr/local/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h:893:77: error: no member named 'erase_all_duplicates' in namespace 'CGAL::internal_np'
  const bool erase_all_duplicates = choose_param(get_param(np, internal_np::erase_all_duplicates), false);
/usr/local/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h:894:73: error: no member named 'require_same_orientation' in namespace 'CGAL::internal_np'
  const bool same_orientation = choose_param(get_param(np, internal_np::require_same_orientation), false);
```

## Source Code

https://github.com/CGAL/cgal/blob/releases/CGAL-4.14-branch/Polygon_mesh_processing/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): OSX 10.14
* Compiler: Apple LLVM version 10.0.1 (clang-1001.0.46.4) 
* Release or debug mode: release
* Specific flags used (if any):
* CGAL version: 4.14
* Boost version: 
* Other libraries versions if used (Eigen, TBB, etc.):

It would appear to me that including the missing directive from #3882 yields further compilation errors in the repair_polygon_soup_example:

## Compilation Errors

In file included from /Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:4:
In file included from /usr/local/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h:26:
/opt/local/include/CGAL/boost/graph/parameters_interface.h:22:26: error: unknown type name 'vertex_point_t'; did you mean 'CGAL::vertex_point_t'?
CGAL_add_named_parameter(vertex_point_t, vertex_point, vertex_point_map)
                         ^
/opt/local/include/CGAL/boost/graph/properties.h:106:14: note: 'CGAL::vertex_point_t' declared here
using boost::vertex_point_t;
             ^
In file included from /Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:4:
In file included from /usr/local/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h:26:
/opt/local/include/CGAL/boost/graph/parameters_interface.h:22:42: error: C++ requires a type specifier for all declarations
CGAL_add_named_parameter(vertex_point_t, vertex_point, vertex_point_map)
                                         ^
/opt/local/include/CGAL/boost/graph/parameters_interface.h:22:56: error: unknown type name 'vertex_point_map'; did you mean 'CGAL::vertex_point_t'?
CGAL_add_named_parameter(vertex_point_t, vertex_point, vertex_point_map)
                                                       ^
/opt/local/include/CGAL/boost/graph/properties.h:106:14: note: 'CGAL::vertex_point_t' declared here
using boost::vertex_point_t;
             ^
In file included from /Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:4:
In file included from /usr/local/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h:26:
/opt/local/include/CGAL/boost/graph/parameters_interface.h:23:1: error: expected function body after function declarator
CGAL_add_named_parameter(halfedge_index_t, halfedge_index, halfedge_index_map)
^
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:12:9: error: use of undeclared identifier 'K'
typedef K::Point_3                                              Point_3;
        ^
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:25:20: error: excess elements in scalar initializer
  points.push_back(Point_3(0,0,0));
                   ^        ~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:26:20: error: excess elements in scalar initializer
  points.push_back(Point_3(1,0,0));
                   ^        ~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:27:20: error: excess elements in scalar initializer
  points.push_back(Point_3(0,1,0));
                   ^        ~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:28:20: error: excess elements in scalar initializer
  points.push_back(Point_3(-1,0,0));
                   ^         ~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:29:20: error: excess elements in scalar initializer
  points.push_back(Point_3(0,-1,0));
                   ^        ~~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:30:20: error: excess elements in scalar initializer
  points.push_back(Point_3(0,1,0)); // duplicate point
                   ^        ~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:31:20: error: excess elements in scalar initializer
  points.push_back(Point_3(0,-2,0)); // unused point
                   ^        ~~~~~
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:70:8: error: no member named 'repair_polygon_soup' in namespace
      'CGAL::Polygon_mesh_processing'
  PMP::repair_polygon_soup(points, polygons);
  ~~~~~^
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:71:8: error: no member named 'orient_polygon_soup' in namespace
      'CGAL::Polygon_mesh_processing'
  PMP::orient_polygon_soup(points, polygons);
  ~~~~~^
/Users/dom/Desktop/CGAL-4.14/examples/Polygon_mesh_processing/repair_polygon_soup_example.cpp:74:8: error: no member named 'polygon_soup_to_polygon_mesh' in namespace
      'CGAL::Polygon_mesh_processing'
  PMP::polygon_soup_to_polygon_mesh(points, polygons, mesh);
  ~~~~~^
15 errors generated.
make[2]: *** [CMakeFiles/repair_polygon_soup_example.dir/repair_polygon_soup_example.cpp.o] Error 1
make[1]: *** [CMakeFiles/repair_polygon_soup_example.dir/all] Error 2
make: *** [all] Error 2

--- a/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h
+++ b/Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair_polygon_soup.h
@@ -23,6 +23,7 @@
 
 #include <CGAL/license/Polygon_mesh_processing/repair.h>
 
+#include <CGAL/boost/graph/named_function_params.h>
 #include <CGAL/Polygon_mesh_processing/internal/named_function_params.h>
 #include <CGAL/Polygon_mesh_processing/internal/named_params_helper.h>
 

